grails-app/controllers/au/org/emii/aatams/AbstractController.groovy

```diff
@@ -21,16 +21,22 @@ class AbstractController
 
         flattenParams()
 
+        doCount(resultList, queryName)
```

Comment:
Cool, I think this is a good way to override the behaviour in the sub class.

I don't think this method needs to follow the do*X*() naming scheme (the methods that do are for actions that display one of the pages, and this is more of just one sub-step). I think something like displayCountMessage() would be good.